src/Components/MyThoughts.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import styled from 'styled-components'
+import { OuterWrapper, InnerWrapper, P, A, ReadMoreArrow } from '../Styled components/GlobalStyles'
+import { WhitePinkH2 } from '../Styled components/HeadlineStyles'
+import { MediumArticles } from '../Data/MediumAricles'
+import VisualLearningImg from '../Images/Medium-Visual-Learning.jpeg'
+
+const MyThoughts = () => {
+  return (
+    <OuterWrapper>
+      <InnerWrapper>
+        <WhitePinkH2>MY THOUGHTS</WhitePinkH2>
+        {MediumArticles.map((article) => {
+          return (
+            <div key={article.name}>
+              <ArticleImg src={VisualLearningImg} alt="" />
+              <Date>{article.date}</Date>
+              <BlogName>{article.name}</BlogName>
+              <A href={article.url}>
```

Comment:
This is not so much a code comment, your code is as always neat and clear. But I loved your text and the choice of picture is excellent! You should absolutely write more pieces like this and post on linkedIN!!